telegraf/1.27/entrypoint.sh

```diff
@@ -11,5 +11,5 @@ else
     # Allow telegraf to send ICMP packets and bind to privliged ports
     setcap cap_net_raw,cap_net_bind_service+ep /usr/bin/telegraf || echo "Failed to set additional capabilities on /usr/bin/telegraf"
 
-    exec setpriv --reuid telegraf --init-groups "$@"
+    exec setpriv --reuid telegraf --regid telegraf --reset-env --clear-groups "$@"
```

Comment:
I know I recommended `--clear-groups`, but let's make the intent clearer and use `--groups telegraf` instead (the end result is the same). Ie:
```
 exec setpriv --reuid telegraf --regid telegraf --groups telegraf "$@"
```

(see discussion about `--reset-env` in another comment)